hyppo/independence/hhg.py

```diff
@@ -224,3 +317,34 @@ def _pearson_stat(distx, disty):  # pragma: no cover
                     S[i, j] = ((n - 2) * (t12 * t21 - t11 * t22) ** 2) / denom
 
     return S
+
+
+def hoeffdings(x, y):
```

Comment:
try using `jit` as here: https://github.com/neurodata/hyppo/blob/f7a1f6c3ba214884c7f508ee60159b99962a3e9c/hyppo/independence/dcorr.py#L293

Reply:
Apparently JIT does not recognize with the rankdata function, so I had to further breakdown the hoeffding function and instead isolate the for-loop as a separate function (_hoeffdings_d_calc). With some minor adjustments, this separate function can be run by Numba and I found good improvements overall in terms of speed!

Something wierd is happening with Codecov however, where it seems to think that _hoeffdings_d_calc is not covered by tests, even though it should have been used in all the tests. Seeking advice on this!